ci/builders/README.md

```diff
@@ -154,231 +143,237 @@ An archive component is used to tell the recipes which artifacts are generated b
 
-Generators can be written in any language but they require to follow some guidance to make them compatible with the engine build system.
+Generators can be written in any language but they require to follow some
+guidelines to make them compatible with the engine build system.
 
 The guidelines are as follows:
 
-* Flags receiving paths to resources from multiple sub-builds need to use paths relative to the checkout directory. If there are global generators in a build configuration, the engine\_v2 recipes will download the full archives of the sub-builds to the checkout/out/&lt;sub-build name> directory.
-* Flags receiving paths to output directories must use paths relative to the out folder. This is to be able to reference the artifacts in the global archives section.
-* The script is in charge of generating the final artifact, e.g. if the script generates multiple files that will be zipped later then it is the script responsibility to generate the final zip.
-* If the generator is producing a Mac/iOS artifact then it is the script responsibility to embed the signing metadata.
-
-Generators contain a single property “tasks” which is a list of tasks to be performed.
-
-```
+* Flags receiving paths to resources from multiple sub-builds need to use paths
+relative to the checkout directory. If there are global generators in a build
+configuration, the engine\_v2 recipes will download the full sub-build archives
+to the checkout/out/&lt;sub-build name> directory.
+* Flags receiving paths to output directories must use paths relative to the out
+folder. This is to be able to reference the artifacts in the global archives
+section.
+* The script is in charge of generating the final artifact, e.g. if the script
+generates multiple files that will be zipped later then it is the script
+responsibility to generate the final zip.
+* If the generator is producing a Mac/iOS artifact then it is the script
+responsibility to embed the signing metadata.
+
+Generators contain a single property “tasks” which is a list of tasks to be
+performed.
+
+```yaml
 "generators": {
-        "tasks": []
+    "tasks": []
  }
 ```
 
-The example above represents a generator configuration with an empty list of tasks.
+The example above represents a generator configuration with an empty list
+of tasks.
 
 ##### Task
 
 Task is a dictionary describing the scripts to be executed.
 
-The properties description is as follows:
+The property's description is as follows:
 
-* **Name**, the name of the step running the script.
-* **Parameters**, flags passed to the script. Both input and output paths must be relative to the checkout directory.
+* **Name** - the name of the step running the script.
+* **Parameters** - flags passed to the script. Both input and output paths must
+be relative to the checkout directory.
 * **Script**, the script path relative to the checkout repository.
 * **Language**, the script language executable to run the script. If empty it is assumed to be bash.
```

Comment:
The bit about the default being `bash` was missing from the description of the `language` field above, I think.

Reply:
Added the default value description to the tests section.